constants/fokabotCommands.py

```diff
@@ -1424,20 +1537,25 @@ async def getPlaytime(fro: str, chan: str, message: list[str]) -> str:
 @command(
     trigger="!whitelist",
     privs=privileges.ADMIN_MANAGE_USERS,
-    syntax="<target_name> <bit>",
+    syntax="<target_name> <reason> <bit>",
```

Comment:
reason if variably lengthed (can be multiple arguments), so must come last

bit is always one argument

Reply:
maybe probably perhaps fixed in cbff5f1a76b376a73512e4f79fcd6d179d59e954